### PR TITLE
git-series 0.8.10 (new formula)

### PR DESCRIPTION
**Pull request by pborzenkov**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

git series tracks changes to a patch series over time. git series also
tracks a cover letter for the patch series, formats the series for
email, and prepares pull requests.

Signed-off-by: Pavel Borzenkov pavel.borzenkov@gmail.com

### CODE REVIEW

File: Formula/git-series.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on "libssh2"
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/git-series"
+    man1.install "git-series.1"
+  end
+
+  test do
+    system bin/"git-series", "-V"
```
Comment (bfontaine) [inline Formula/git-series.rb]: Can’t we write a test that does something more substantial than e.g. `-V`, `--version`, or `--help`?

File: Formula/git-series.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    bin.install "target/release/git-series"
+    man1.install "git-series.1"
+  end
+
+  test do
+    (testpath/".gitconfig").write <<-EOS.undent
+      [user]
+        name = Real Person
+        email = notacat@hotmail.cat
+      EOS
```
Comment (MikeMcQuaid) [inline Formula/git-series.rb]: Tiny nit: can you unindent this by two characters? Thanks!

File: Formula/git-series.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    system "git", "init"
+    (testpath/"test").write "foo"
+    system "git", "add", "test"
+    system "git", "commit", "-m", "Initial commit"
+    (testpath/"test").append_lines "bar"
+    system "git", "commit", "-m", "Second commit", "test"
+    system bin/"git-series", "start", "feature"
+    system "git", "checkout", "HEAD~1"
+    system bin/"git-series", "base", "HEAD"
+    system bin/"git-series", "commit", "-a", "-m", "new feature v1"
```
Comment (MikeMcQuaid) [inline Formula/git-series.rb]: Is it worth checking the output of any of these commands? "No" is an acceptable answer 😁

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/git-series.rb (hunk 3 above)
Comment (pborzenkov) [inline Formula/git-series.rb]: @MikeMcQuaid They don't output anything particularly useful for the test, only that the series's been successfully started/commit. But this is already covered by the return code. So I believe there is no need to check the output.

File: Formula/git-series.rb (hunk 2 above)
Comment (pborzenkov) [inline Formula/git-series.rb]: @MikeMcQuaid Done

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**